ClinGen/clincoded
# Connects to #846. Connects to #885.

This PR consists of implementations pertinent to the change requests in constructing the external ExAC linkout differently when there is no ExAC population data is found for the table display on the Population tab.

**Technical details:**
1. In the `hgvs_notation.js` and `parse-resources.js` files, I added "if" statements to several places to ensure script execution even when the expected data is absent in the REST API response.
2. In the `basic_info.js` file, removed no-longer-used code and refactored code to ensure script execution even when the expected data is absent in the REST API response.

**Steps to test:**
1. Login and use **53981** and/or **90973** for selecting a variant (relevant to #885).
2. Expect to see that the long HGVS terms are ellipsis and not pushing the table beyond the page on the Basic Info tab.
3. Also expect to see that the "asterisk-marked" canonical transcripts are no longer having a tinted red background color on the Basic Info tab (this was missed in my previous PR).
4. The following variants are for testing the external ExAC linkout on the Population tab:
* **55962** - expect the "See ExAC" link going to ExAC variant page.
* **53905, 53205, 54032, 53981** - expect the "See ExAC" link going to ExAC region search.
* **CA501146, CA501097** - expect the "See ExAC" link going to ExAC homepage.
